cleanlab/internal/multilabel_utils.py

```diff
@@ -64,3 +64,34 @@ def get_onehot_num_classes(labels, pred_probs=None):
             "wrong format for labels, should be a list of list[indices], please check the documentation in find_label_issues for further information"
         )
     return y_one, num_classes
+
+
+def int2onehot(labels, K) -> np.ndarray:
+    """Convert list of lists to a onehot matrix for multi-labels
+    Parameters
```

Comment:
```suggestion
    """Convert multi-label classification `labels` from a ``List[List[int]]`` format to a onehot matrix.
    This returns a binarized format of the labels as a multi-hot vector for each example, where the entries in this vector are 1 for each class that applies to this example and 0 otherwise.
    
    Parameters
```

Reply:
note there should always be more spacing between the initial docstring description and `Parameters` or `Returns` or `Note` or any other special header.